src/pages/ProductDetail/FavoriteModal.js

```diff
@@ -26,44 +28,62 @@ const FavoriteModal = ({
     };
   }, []);
 
-  if (!visible) return null;
-
   return (
     <DarkBackground onClick={onConfirm}>
-      <ModalWrapper onClick={e => e.stopPropagation()}>
+      <Wrapper onClick={e => e.stopPropagation()}>
         <ModalHead>
           <h2>{title}</h2>
         </ModalHead>
         <ModalContent>
           <ProductInfo>
             <SuggestItem>
-              <img alt="제품이미지" src={productImg} />
+              <img alt="제품이미지" src={productImg?.url} />
               <span>{productName}</span>
             </SuggestItem>
           </ProductInfo>
           <ProductSize>
-            {productSizes?.map(size => (
-              <MarginButton
-                color="lightGrey"
-                outline
-                fullWidth
-                key={size.id}
-                name={size.name}
-              >
-                <span>{size.name}</span>
-                <span>
-                  <FontAwesomeIcon icon={farBookmark} size="1x" />
-                </span>
-              </MarginButton>
-            ))}
+            {productSizes?.map(size => {
+              return (
+                <MarginBtn
+                  onClick={btnClick}
+                  key={size.id}
+                  name={size.name}
+                  value={size.id}
+                  outline
+                  color="lightGrey"
+                  fullWidth
```

Comment:
```suggestion
                  onClick={btnClick}
                  key={size.id}
                  name={size.name}
                  value={size.id}
                  color="lightGrey"
                  outline
                  fullWidth
```

value 명시적 부여하는 것들이랑, true값만 부여하는 것들 순서 구분지어서 좀 더 읽기좋게 해주는게 좋습니다!

Reply:
반영완료